feat: carbon badge in the footer

broarr opened the pull request:
This commit adds a carbon tracker badge to the footer of the website. This'll be the first step in making a more carbon friendly website. This PR is just the badge, I'll fix the CI stuff at a later point

mcmcgrath13 commented:
@TiffL the test has an error

```
Error: Error building site: failed to render pages: render of "section" failed: "/site/themes/gb-theme/layouts/about/list.html:231:29": execute of template failed: template: about/list.html:231:29: executing "content" at <len .author>: error calling len: len of untyped nil
```

Could you look into this?

mcmcgrath13 commented:
![image](https://user-images.githubusercontent.com/24885580/68961102-88878a00-079f-11ea-894b-d04e13182097.png)

@fernandogelin @TiffL I posit this looks kind of funny - how do we want to address having so many positions open

TiffL commented:
Okay, I've updated to have them match the first row

bradfordbarr committed:
feat: carbon badge in the footer

This commit adds a carbon tracker badge to the footer of the website.
This'll be the first step in making a more carbon friendly website

ashleysyg reviewed (approved):
looks good!

eldu commented on components/Footer.tsx:
```diff
@@ -103,9 +104,12 @@ const Footer = () => {
-        <div>&copy; Brown University</div>
-        <CCVLogo />
+      <div className="content-wrapper flex flex-row items-start justify-between py-4 bg-black font-serif text-base text-cream md:text-lg">
+        <div className="flex flex-col-reverse gap-4 items-start">
```
It looks like the CCV, (C) Brown University, and the carbon calculator are on 2 lines. Perhaps this should be a flex row to put it on one line instead

Did you like the column here more? 
<img width="1609" alt="Screenshot 2025-06-03 at 1 36 36 PM" src="https://github.com/user-attachments/assets/a6de65a2-fd6e-4073-9bff-d76465b1d1b0" />

broarr replied:
The carbon calculator has a second line. When I make time to fix the CI you'll be able to see it

eldu replied:
Oohh okay cool cool. Well then i think it's all good to go then !

eldu reviewed (approved):
LGTM! I added in a comment
